ggshield/scan/scanner.py

```diff
@@ -245,17 +246,25 @@ def _start_scans(
         chunks_for_futures = {}
         skipped_chunk = []
 
-        chunk: List[File] = []
-        for file in files:
-            if file.document:
-                chunk.append(file)
+        chunk: List[Scannable] = []
+        for scannable in scannables:
+            if scannable.is_longer_than(DOCUMENT_SIZE_THRESHOLD_BYTES):
+                click.echo(
+                    f"ignoring file over {DOCUMENT_SIZE_THRESHOLD_BYTES // 1024 // 1024} MB:"
+                    f" {scannable.path}",
+                    err=True,
+                )
+                continue
```

Comment:
This scannable won't be part of the skipped_chunk then ?

Reply:
Edit : I think this gets fixed in another commit.

Reply:
Yes, sorry for that. The history in this PR could have used more cleanups.